tests/test_ditto_cli.py

```diff
@@ -11,19 +12,18 @@
 
 current_directory = os.path.dirname(os.path.realpath(__file__))
 
-
 def test_opendss_to_gridlabd_cli():
 
     output_path = tempfile.TemporaryDirectory()
-    print(output_path.name)
-    p = subprocess.Popen(
+
+    run_command(
         shlex.split(
-            """ ditto-cli convert --from="opendss" --to="gridlabd" --input="./tests/data/small_cases/opendss/ieee_13node/master.dss" --output="{}" """.format(
+            """ poetry run ditto-cli convert --from="opendss" --to="gridlabd" --input="./tests/data/small_cases/opendss/ieee_13node/master.dss" --output="{}" """.format(
```

Comment:
`poetry run` is now necessary (on windows only) for these commands to work (from within the unit tests only)

Reply:
subprocess Popen is really flexible and also we can control over the processes to work with. Just to be aware that it can make slower when many files are open or does all sort of checking in cross-platform support.

Reply:
The subprocess call is just a quick (existing) unit test to make sure the ditto-cli entrypoint works, so no performance issues here.